lib/stimulus_reflex/channel.rb

```diff
@@ -100,8 +101,15 @@ def render_page(url, reflex)
   def broadcast_morphs(selectors, data, html)
     document = Nokogiri::HTML(html)
     selectors.each do |selector|
-      html = document.css(selector).inner_html
-      cable_ready[stream_name].morph selector: selector, html: html, children_only: true, permanent_attribute_name: "data-reflex-permanent", stimulus_reflex: data
+      match = document.css(selector)
+      next if match.blank?
```

Comment:
I really appreciate that you're being liberal in what you accept... not taking for granted that the data coming from the client will be well-formed, even though we also remove empty and invalid selectors in the client library.

However, within this function it's not clear to me how match could ever be blank given line 22 you say selectors = ["body"] if selectors.blank?

Reply:
It’s possible that the newly rendered page doesn’t include the matched element after the reflex makes state changes. It’s an edge case but is certainly plausible.